spec/requests/product_drives_requests_spec.rb

```diff
@@ -116,6 +117,23 @@
           expect(response.body).not_to include('early_product_drive')
           expect(response.body).not_to include('late_product_drive')
         end
+
+        it "returns the quantity of all organization's items" do
+          product_drive = create(:product_drive, name: 'product_drive', organization: organization)
+
+          items = organization.items
+
+          donation = create(:product_drive_donation, product_drive: product_drive)
+          create(:line_item, :donation, itemizable_id: donation.id, item_id: items.first.id, quantity: 4)
+
+          subject
+
+          row = response.body.split("\n")[1]
+          expect(response.body).to include(items.first.name)
+          expect(response.body).to include(items.second.name)
+          expect(row.split(',')[7]).to eq('4')
+          expect(row.split(',')[8]).to eq('0')
+        end
       end
```

Comment:
I'd also like to have a demonstration that any inactive items on the organization are being included.